bin/improver-tests

```diff
@@ -175,17 +187,22 @@ Optional arguments:
 
 Arguments:
     SUBTEST         Name(s) of a subtest to run without running the rest.
-                    Valid names are: pycodestyle, pylint, pylintE, licence, doc, unit, cli.
-                    pycodestyle, pylintE, licence, doc, unit, and cli are the default tests.
-    SUBCLI          Name(s) of cli subtests to run without running the rest.
-                    Valid names are tasks which appear in /improver/tests/
-                    without the "improver-" prefix. The default is to run all
-                    cli tests in the /improver/tests/ directory.
-                    e.g. 'improver tests cli nbhood' will run neighbourhood
-                    processing cli tests only.
+                    Valid names are: pycodestyle, pylint, pylintE, licence,
+                    doc, unit, cli. pycodestyle, pylintE, licence, doc, unit,
+                    and cli are the default tests.
+    SUBCLI          Name(s) of cli tests to run without running the rest.
+                    Valid names are either directory names which appear in
+                    tests/ minus the "improver-" prefix, or paths to individual
+                    BATS test files. The default is to run all cli tests in the
+                    tests/ directory. For example, 'improver tests cli nbhood'
+                    will run neighbourhood processing cli tests in
+                    tests/improver-nbhood/*.bats.
+                    'improver tests cli ~/improver/tests/improver-nbhood/01-help.bats'
+                    will only run that specific nbhood CLI test.
```

Comment:
Note that, whether by design or accident, you can also run single tests in the following way:

```
improver tests cli nbhood/05-mask-square.bats
```
This is worth including in the examples here.